[dynamo,easy] Add Typing variable to possible dict keys

lezcano opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #118630
* #118855
* #118535
* #118199
* #118208
* __->__ #118003
* #118194
* #117625
* #117983
* #118098
* #117982

With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

lezcano committed:
Add Typing variable to possible dict keys

With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

[ghstack-poisoned]

anijain2305 reviewed (approved):
Possible to add a test?

lezcano commented:
Should fix a test that was breaking at https://hud.pytorch.org/pytorch/pytorch/pull/117631?sha=f76cfff70eb15a74e0abbcc6d2da38072659879f. But sure, let me add a test.

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

lezcano committed:
Update on "[dynamo,easy] Add Typing variable to possible dict keys"


With this one, the only keys we are not tracing properly in the
(non-skipped) test suite are `OutDtypeHigherOrderVariable()`, and a
couple `UserDefinedObjectVariables`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]
